packages/schematics/angular/migrations/update-14/update-angular-config.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Rule } from '@angular-devkit/schematics';
+import { allTargetOptions, updateWorkspace } from '../../utility/workspace';
+
+/** Migration to remove deprecated/removed builder options. */
+export default function (): Rule {
+  return updateWorkspace((workspace) => {
+    for (const project of workspace.projects.values()) {
+      for (const target of project.targets.values()) {
+        if (
+          target.builder === '@angular-devkit/build-angular:server' ||
+          target.builder === '@angular-devkit/build-angular:browser'
+        ) {
+          for (const [, options] of allTargetOptions(target)) {
```

Comment:
Nit: Can use `.values()` / `Object.values()` to drop the destructure.

Reply:
Sorry about this, old habit!

Reply:
No problem! I don't have anything against empty destructures, just think we should use an API that directly gives what we want when it's possible.